CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/Server.java

```diff
@@ -210,16 +209,26 @@ else if ("flushMemory".equals(method)) {
                 }
                 else {
                     result = target.invoke(requestHandler, args);
-                    body = ValueSerializer.serialize(result, memory);
+                    body = (result instanceof RawData)
+                        ? result
+                        : ValueSerializer.serialize(result, memory);
                 }
             }
             session.getHeaders();
-            if (body != null) {
-                IStatus status = Status.OK;
-                return Response.newFixedLengthResponse(status, "text/plain", body.getBytes());
+            if (body == null) {
+                return Response.newFixedLengthResponse(Status.OK, "text/plain", "I-1");
             }
             else {
```

Comment:
this else is not needed, because if body==null already returns. anyway, this is optional, nice to make the logic simple

Reply:
agree.